### PR TITLE
Port step processing and download all subs functionality.

### PR DESCRIPTION
**Pull request by kjohnson**

### CODE REVIEW

**Comment by klhall1987** [comment]
@kjohnson I'm not seeing an option to export all forms.

**Comment by kjohnson** [comment]
Download all "Submissions".

**Comment by klhall1987** [comment]
Sorry subs is what I meant. Here is a screen shot of what I'm seeing on my subs page. 

Screenshot: 
screen shot 2016-08-30 at 11 56 48 am

**Comment by kjohnson** [comment]
Are you on the right branch? Also, check the console, as this is added with JS.
screenshot 2016-08-30 11 59 07

File: includes/Admin/CPT/DownloadAllSubmissions.php [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                    },5000);
+                    <?php
+                    }
+
+                    if ( isset ( $_REQUEST['form_id'] ) && ! empty ( $_REQUEST['form_id'] ) ) {
+                    $redirect = urlencode( remove_query_arg( array( 'download_all', 'download_file' ) ) );
+                    $url = admin_url( 'admin.php?page=nf-processing&action=download_all_subs&form_id=' . absint( $_REQUEST['form_id'] ) . '&redirect=' . $redirect );
+                    $url = esc_url( $url );
+                    ?>
+                    var button = '<a href="<?php echo $url; ?>" class="button-secondary nf-download-all"><?php echo __( 'Download All Submissions', 'ninja-forms' ); ?></a>';
```
Comment (kjohnson) [inline includes/Admin/CPT/DownloadAllSubmissions.php]: Here is where the button is added.

**Comment by klhall1987** [comment]
Okay I found the problem. It works if you go submissions and then select the form via the drop down menu. However if you click view submissions on the all forms page, the "download all submissions" button doesn't appear.

**Comment by klhall1987** [comment]
Tested and working correctly passing to @kstover for merge.

**Pull request merged**